src/services/SubmissionService.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export class SubmissionService {
+
+    constructor() {
+        this.axios = axios.create({
+            baseURL: process.env.REACT_APP_PROBLEM_SVC_BASE_URL,
+            timeout: 10000
+        });
+    }
+
+    async submit(problem) {
+        const {problemId, langEnvName, studentId} = problem
```

Comment:
wtf is this? `const {problemId, langEnvName, studentId} = problem`

please don't rush on the code.
Ask @snowmancc for the api details (rejudge)